schema/deploy/queries/facility_application_by_reporting_year.sql

```diff
@@ -25,8 +25,8 @@ create or replace function ggircs_portal.facility_application_by_reporting_year(
       on facility.id = application.facility_id
       and application.reporting_year = (select reporting_year from ggircs_portal.default_displayed_reporting_year())
     join ggircs_portal.organisation on facility.organisation_id = organisation.id
-    where _reporting_year is null
-    order by organisation.operator_name, facility.facility_name);
+    where _reporting_year is null)
+  ) sub order by operator_name, facility_name;
```

Comment:
what does this 'sub' do?

Reply:
it's just an alias for the subquery, which is required my postgres